lib/nested_filter.ex

```diff
@@ -5,57 +5,45 @@ defmodule NestedFilter do
 
   @doc """
   Take a (nested) map and filter out any keys with specified values in the
-  filter_values list.
+  values_to_reject list.
   """
-  @spec drop_by_value(map :: map(), filter_values :: list()) :: map()
```

Comment:
In elixir, `filter` _keeps_ things and `reject` _removes_ them. Naming updated to reflect that (and you should really just use those functions).